plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -943,6 +962,11 @@ public boolean configure(final String name, final Map<String, Object> params) th
             _mountPoint = "/mnt";
         }
 
+        _libvirtConnectionProtocol = (String) params.get("libvirt.connection.protocol");
+        if (_libvirtConnectionProtocol == null) {
+            _libvirtConnectionProtocol = "qemu://";
```

Comment:
It's something I have to document in the installation doc. If you want to run the migration over TLS, you have to specify here `qemu+tls://`